2023/incoming/perl-the-humane-programming-language.pod

```diff
@@ -0,0 +1,90 @@
+Author: perl@dnmfarrell.com
+Title: Perl - The Humane Programming Language
+Topic: 
+
+state of the onion first three talks
+people are different
+
+=encoding utf8
+
+=head2 Perl - The Humane Programming Language
+
+Human languages follow the L<Law of Brevity|https://en.wikipedia.org/wiki/Brevity_law>, where the most commonly used words are also the shortest. Perl aims for brevity too. Its keywords tend to be shorter than the traditional versions:
```

Comment:
I'd call out Zipf directly and his book about it. Details like that give nice flavor to the prose.